application/models/Logbook_model.php

```diff
@@ -151,12 +157,50 @@ function create_qso() {
       $clean_county_input = null;
     }
 
-    if($this->input->post('copyexchangetodok')) {
-      $darc_dok = $this->input->post('exch_rcvd');
-    } else {
-      $darc_dok = $this->input->post('darc_dok');
+    $darc_dok = trim(xss_clean($this->input->post('darc_dok')))
```

Comment:
The trailing semicolon is missing. Leading to a PHP error.
Fix in: https://github.com/DJ3CE/wavelog/pull/1

Reply:
Please accept my apologies - both the missing bracket and this 'incident' happened, because I did retype the changes between my 'pushing machine' and the 'testing machine'...

Reply:
No worries. Tnx for fix. Now on with testing here.